packages/stacked_generator/lib/src/generators/forms/stacked_form_content_generator.dart

```diff
@@ -1,262 +1,41 @@
-import 'package:recase/recase.dart';
-import 'package:stacked_generator/src/generators/base_generator.dart';
-import 'package:stacked_generator/src/generators/forms/field_config.dart';
+import 'package:stacked_generator/src/generators/forms/form_util_service.dart';
 import 'package:stacked_generator/src/generators/forms/form_view_config.dart';
 
 /// A generator that creates the form code based on the configs passed in
-class StackedFormContentGenerator extends BaseGenerator {
-  final FormViewConfig _formViewConfig;
-
-  StackedFormContentGenerator(this._formViewConfig);
+class StackedFormContentGenerator extends FormUtilService {
+  final FormViewConfig formViewConfig;
+  StackedFormContentGenerator(this.formViewConfig)
+      : super(formViewConfig: formViewConfig);
 
   String generate() {
     writeLine("// ignore_for_file: public_member_api_docs");
 
-    final fields = _formViewConfig.fields;
-
     // TODO: All of these functions can be moved into a separate util class or service
     // and can then be unit tested to avoid simple mistakes. BUT, that's for later.
-    _generateImports();
-    _generateValueMapKeys(fields);
-    _generateDropdownItemsMap(fields.onlyDropdownFieldConfigs);
+    generateImports();
+    generateValueMapKeys();
+    generateDropdownItemsMap();
+    generateTextEditingControllerItemsMap();
     _generateFormMixin();
-    _generateFormViewModelExtensions(fields);
+    _generateFormViewModelExtensions();
```

Comment:
Mixing the private and public method is a bad practice 
I prefere if you revert to extend the `BaseGenerator` and use the `FormUtilService` as a helper class